docs/extensions/readme.md

```diff
@@ -350,4 +352,58 @@ LongRunningOperationRetryTimeout | `int` | Retry timeout in seconds for Long Run
     "AcceptLanguage": "EN-GB",
     "LongRunningOperationRetryTimeout": 9001
 }
-```
\ No newline at end of file
+```
+
+## x-ps-client-side-filter
+Defines client-side filters
+
+**Parent element**: `x-ps-cmdlet-info Object`
+
+**Schema**:
+
+Field Name | Type | Description
+---|:---:|---
+serverSideResultCommand | `string` | Cmdlet to run to get server-side results. Use "." to indicate the current cmdlet.
+serverSideResultParameterSet | `string` | Parameter set to get server-side results.
+clientSideParameterSet | `string` | Parameter set to get parameters from.
+filters | `x-ps-client-side-filter-definition[]` | Filters
+
+**Examples**:
+```json5
+{
+    "serverSideResultCommand": ".",
+    "serverSideResultParameterSet": "BatchAccount_List",
+    "clientSideParameterSet": "BatchAccount_Get",
+    "filters": ...
+}

```

Comment:
Currently, I see only one element is allowed.
Should we support an array of x-ps-client-side-filter in x-ps-cmdlet-info Object for providing this wild card support for other clientSideParameterSet parameter sets?

Reply:
f82a3f7